GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -535,14 +537,14 @@ private void FileTreeContextMenu_Opening(object sender, System.ComponentModel.Ca
         {
             var gitItem = tvGitTree.SelectedNode?.Tag as GitItem;
             var itemSelected = gitItem is not null;
-            var isFile = itemSelected && gitItem.ObjectType == GitObjectType.Blob;
-            var isFolder = itemSelected && gitItem.ObjectType == GitObjectType.Tree;
+            var isFile = gitItem?.ObjectType == GitObjectType.Blob;
+            var isFolder = gitItem?.ObjectType == GitObjectType.Tree;
             var isFileOrFolder = isFile || isFolder;
 
             // Many items does not make sense if a local file does not exist, why this is used for Enabled
-            var isExistingFileOrDirectory = itemSelected && FormBrowseUtil.IsFileOrDirectory(_fullPathResolver.Resolve(gitItem.FileName));
+            var isExistingFileOrDirectory = gitItem is not null && FormBrowseUtil.IsFileOrDirectory(_fullPathResolver.Resolve(gitItem.FileName));
```

Comment:
The variable is still available, why change this only?

Reply:
Roslyn doesn't (currently) track the fact that the bool `itemSelected` implies that `gitItem` is not null.